react/features/remote-control/components/RemoteControlAuthorizationDialog.js

```diff
@@ -0,0 +1,165 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import {
+    Dialog,
+    hideDialog
+} from '../../base/dialog';
+import { translate } from '../../base/i18n';
+import { getParticipantById } from '../../base/participants';
+
+declare var APP: Object;
+
+/**
+ * Implements a dialog for remote control authorization.
+ */
+class RemoteControlAuthorizationDialog extends Component {
+    /**
+     * RemoteControlAuthorizationDialog component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * The display name of the participant who is requesting authorization
+         * for remote desktop control session.
+         *
+         * @private
+         */
+        _displayName: React.PropTypes.string,
+
+        /**
+         * Used to show hide the dialog on cancel.
+         */
+        dispatch: React.PropTypes.func,
+
+        /**
+         * The ID of the participant who is requesting authorization for remote
+         * desktop control session.
+         *
+         * @public
+         */
+        participantId: React.PropTypes.string,
+
+        /**
+         * Invoked to obtain translated strings.
+         */
+        t: React.PropTypes.func
+    }
+
+    /**
+     * Initializes a new RemoteControlAuthorizationDialog instance.
+     *
+     * @param {Object} props - The read-only properties with which the new
+     * instance is to be initialized.
+     */
+    constructor(props) {
+        super(props);
+
+        this._onCancel = this._onCancel.bind(this);
+        this._onSubmit = this._onSubmit.bind(this);
```

Comment:
Yes I'm sure of it. If you have the authorization dialog open and press t the modal content will switch. But maybe the fix then is to prevent other atlaskit modals from opening while one is already open, which would be for another pull request.